Sat Sep 16 14:35:55 2000  Richard Kenner  <kenner@vlsi1.ultra.nyu.edu>

	* stor-layout.c (place_field): Don't do anything if field or
	its type are ERROR_MARK.

--- gcc/stor-layout.c	2000/08/29 21:39:45	1.83
+++ gcc/stor-layout.c	2000/09/16 18:31:00	1.84
@@ -638,6 +638,9 @@ place_field (rli, field)
   /* The type of this field.  */
   tree type = TREE_TYPE (field);
  
+  if (TREE_CODE (field) == ERROR_MARK || TREE_CODE (type) == ERROR_MARK)
+      return;
+
   /* If FIELD is static, then treat it like a separate variable, not
      really like a structure field.  If it is a FUNCTION_DECL, it's a
      method.  In both cases, all we do is lay out the decl, and we do
